pennylane/ops/qubit/observables.py

```diff
@@ -331,16 +331,17 @@ def compute_sparse_matrix(H):  # pylint: disable=arguments-differ
 
 class Projector(Observable):
     r"""
-    Observable corresponding to the computational basis state projector :math:`P=\ket{i}\bra{i}`.
+    Observable corresponding to the state projector :math:`P=\ket{\phi}\bra{\phi}`.
 
     The expectation of this observable returns the value
 
     .. math::
-        |\langle \psi | i \rangle |^2
+        |\langle \psi | \phi \rangle |^2
 
-    corresponding to the probability of measuring the quantum state in the :math:`i` -th eigenstate of the specified :math:`n` qubits.
+    corresponding to the probability of measuring the quantum state :math:`\psi` in the basis defined by :math:`\phi`.
 
-    For example, the projector :math:`\ket{11}\bra{11}` , or in integer notation :math:`\ket{3}\bra{3}`, is created by ``basis_state=np.array([1, 1])``.
+    For example, the projector :math:`\ket{11}\bra{11}` (:math:`\ket{3}\bra{3}` in integer notation) is created by ``basis_state=np.array([1, 1])``,
+    or the projector :math:`\ket{+}\bra{+}` is created by ``state_vector=np.array([1, 1])/np.sqrt(2)``.
```

Comment:
We could turn this into an actual example, i.e., with one or more code blocks like [here](https://docs.pennylane.ai/en/stable/code/api/pennylane.FermionicSWAP.html).

Reply:
I agree! I've replaced this by a code example with two projectors. Let me know what you think :)